### ISSUE TITLE
Paste Bug

### ISSUE THREAD

**Issue by 3990995** [jessesquires/JSQMessagesViewController#1836]
## New issue checklist

- [x] I have read all of the [`README`](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/README.md), [documentation](http://cocoadocs.org/docsets/JSQMessagesViewController/), and [FAQ](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/Documentation/faq.md).
- [x] I have reviewed the [contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md). Confirmation: ____
- [x] I have searched [existing issues](https://github.com/jessesquires/JSQMessagesViewController/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

### General information

- Library version(s):
- iOS version(s):
- Devices/Simulators affected:
- Reproducible in the demo project? (Yes/No): 
- Related issues:

### Crash log? Screenshots? Videos? Sample project?

```
if ([messageData isMediaMessage]) {
            id<JSQMessageMediaData> mediaData = [messageData media];
            if ([messageData respondsToSelector:@selector(mediaDataType)]) {
//永远不会执行
                [[UIPasteboard generalPasteboard] setValue:[mediaData mediaData]
                                         forPasteboardType:[mediaData mediaDataType]];
            }
        }
```

**Comment by Lucashuang0802** [comment]
Thanks! 😄

Please provide all the information in the issue template so we can help! 
And don't forget to read our contribution guidelines. 😊

**Comment by 3990995** [comment]
运行demo，copy一张图片，if语句不会通过执行。图片不会被复制。
- (void)collectionView:(JSQMessagesCollectionView *)collectionView performAction:(SEL)action forItemAtIndexPath:(NSIndexPath *)indexPath withSender:(id)sender；

修改成下面的判断就可以了  ，change messagesData to mediaData is ok
if ([mediaData respondsToSelector:@selector(mediaDataType)]) {
                [[UIPasteboard generalPasteboard] setValue:[mediaData mediaData]
                                         forPasteboardType:[mediaData mediaDataType]];
            }

**Comment by Lucashuang0802** [comment]
@3990995 it's fixed on #1871. Thanks for reporting 😃
